### ISSUE TITLE
Refactor the Views domain to match the proposed architecture

### ISSUE THREAD

**Issue by reyraa** [LiskHQ/lisk-desktop#4202]
### Description

In order to make all the other domains perform correctly, the View domain should be able to render routes correctly, and provide each domain with necessary data. Although the details of the implementation of each view can be changed according to the technical needs of each corresponding domain component, the basic implementation must be there.

Additionally, the basic components and shared components must be defined and implemented for the other domains to use.

### Acceptance Criteria
- The components that were commented out should be uncommented. 
- The property drilling issue must be fixed by the use of React context API.
- The router should perform correctly, although the domains are missing and the page does not render correctly.
- The basic components must be in the right place and have the right naming. If refactoring is foreseen, at least a note/Github ticket must be made.
- The shared components must be in the right place and have the right naming. If refactoring is foreseen, at least a note/Github ticket must be made.
- Each view must make the required API calls and import the required child components from corresponding domains. Although the correct implementation of said components are postponed to their dedicated tickets.

### Additional information 
The goal of this ticket is to make the views ready for the other tickets and define the required job for refactoring basic and shared components. Once this ticket is done, we should be able to refactor each domain individually and independently, we should also know if a shared or basic component needs some refactoring to work as desired.
